packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -434,8 +434,10 @@ class PostsRepo extends AbstractRepo<"Posts"> {
       LEFT JOIN "Revisions" r ON r."_id" = p."contents_latest"
       LEFT JOIN "PostEmbeddings" pe ON p."_id" = pe."postId"
       WHERE
+        ${getViewablePostsSelector('p')} AND
         pe."embeddings" IS NULL AND
-        COALESCE((r."wordCount")::INTEGER, 0) > 0
+        COALESCE((r."wordCount")::INTEGER, 0) > 1 AND
+        r.html NOT LIKE '%|endoftext|%'
```

Comment:
I think the fix that removes the weird token is better than excluding that set of posts